HOW/social/process/how/how-v1.md

```diff
@@ -0,0 +1,164 @@
+---
+HOW: /social/process/how
+Source: /social/process/how/how-v1.md
+Status: Draft
+Created: 2024-11-28
+Version: idx/1
+---
+
+## Title
+HOW Default Process
+
+## Author(s)
+
+zippy <eric@holochain.org>, mattyg <matt@buildyourweb.app>
+
+## Abstract
+
+This HOW describes a default process, format and workflow for how HOWs come into being. HOWs are the primary mechanism for proposing new features, collecting community input on an issue, and documenting decisions affecting Holochain and/or standards and processes used by Holochain hApps or by the Holochain ecosystem.  In other words, this HOW describes **how** the community comes into alignment.
+
+## Motivation
+
+As Holochain and the Holochain ecosystems evolves, we want a standardized process for proposing, adopting, and documenting standards of all types, including: protocol creation and changes, feature additions, and process improvements, etc. A well-defined HOW process helps:
+- Maintain transparency in decision-making
+- Ensure proper technical documentation
+- Foster community participation
+- Track the history of protocol changes
+- Enable structured discussions about significant changes
+- Enable coherent evolution of both Holochain itself, as well as the patterns and processes in use by the wider Holochain ecosystem.
+
+## Rationale
+
+This standard, as the first and the default process for adding HOWs to the HOW tree fully addresses the motivation:
+- as this process is implemented in an open github repo it both maintains transparency of the decisions made and encourages public feedback and comments in the process, as well as tracks all changes made over time.
+- the the HOW tree and the formal process for requirements for parts of the tree and adding to it ensures proper technical documentation as well as creates a clear method for evolving the process itself.
+
+## Specification
+
+### General
+HOW standards are held in a git repo on github, and use a small number of Github's affordances to help with the process.  This backing-store is intended to be replaced by a Holochain app in the future.  The HOW processes and data formats consist of a tree structure that holds all types of standards for the Holochain framework and ecosystem.
+
+### HOW Tree
+
+Rather than using the pattern of storing and identifying standards as a long numerical list (as in RFCs or EIPs), HOWs are identified by a semantic path in a hierarchy.  This allows their relationship and types to be more readily understood.  The HOW tree is stored directly as a file hierarchy in a git repo. The HOW tree encompasses different types of standards where the type is represented as the directory structure of in the tree.  The initial structure is:
```

Comment:
```suggestion
Rather than using the pattern of storing and identifying standards as a long numerical list (as in RFCs or EIPs), HOWs are identified by a semantic path in a hierarchy.  This allows their relationship and types to be more readily understood.  The HOW tree is stored directly as a file hierarchy in a git repo. The HOW tree encompasses different types of standards where the type is represented as the directory structure in the tree.  The initial structure is:
```